neural_lam/models/ar_model.py

```diff
@@ -246,7 +246,7 @@ def validation_step(self, batch, batch_idx):
         # Log loss per time step forward and mean
         val_log_dict = {
             f"val_loss_unroll{step}": time_step_loss[step - 1]
-            for step in constants.VAL_STEP_LOG_ERRORS
+            for step in self.args.val_steps_log
```

Comment:
This is another one I would consider renaming. Maybe `val_steps_to_log`? I think `val_steps_log` is better than `VAL_STEP_LOG_ERRORS` because the latter could either be interpreted as a verb (*log these errors") or the "log errors of val step"